mne/time_frequency/psd.py

```diff
@@ -128,16 +145,21 @@ def psd_array_welch(x, sfreq, fmin=0, fmax=np.inf, n_fft=256, n_overlap=0,
                              for d in x_splits)
 
     # Combining, reducing windows and reshaping to original data shape
-    psds = np.concatenate([np.nanmean(f_s, axis=-1)
-                           for f_s in f_spectrogram], axis=0)
-    psds.shape = np.hstack([dshape, -1])
+    if combine is not None:
```

Comment:
I'm not sure if I understand: if combine is None (no redcution, windows are returned at -1 dimension) then _get_reduction will return an error. Do you suggest I should make _get_reduction return a function that just passes its input when `combine=None`?